### PR TITLE
Added the possibility to use a fixed colWidth

### PR DESCRIPTION
**Pull request by ThinklabsGbR**
When using a fixed grid it is necessary to use a fixed colWidth. Especially when filling the cards array dynamically with objects and random template usages. With that it can happen, that you have no .col1 element inside your cards array and the automatic calculation will result in a wrong min colWidth which will then lead to "Image too large" errors.

### CODE REVIEW

**Comment by tristanguigue** [comment]
I'm not sure I see the use case here, if there are no items to be displayed then this is not a problem. If there are items to display it will pick up the smallest width to set as column width. The width of the items are entirely depends on the CSS. Could you provide an example? Thanks

**Comment by ThinklabsGbR** [comment]
As with masonry it is sometimes necessary than you can specify the colWidth. I had an example where i was filling my cards array dynamically. I was using 4 templates, for col1 - col4 layouts. In one case i only had 2 elements in my array, one col3 and one col4 template and i got this "Item too large" errors and it didn't aligned anything at all. After specifying a fixed colWidth all errors disappeared.

File: src/position.service.js [hunk 1]
```diff
@@ -149,12 +149,12 @@
...
      * @return: the promise of the position animations being completed
      */
-    function layout(containerWidth) {
+    function layout(containerWidth, colWidth) {
       // We first gather the items dimension based on the DOM elements
       items = self.getItemsDimensionFromDOM();
 
       // Then we get the column size base the elements minimum width
-      var colSize = getColSize();
+      var colSize = getColSize(colWidth);
```
Comment (tristanguigue) [inline src/position.service.js]: You can use
```
var colSize = colWidth || getColSize()
```
And leave `getColSize` as it were.

Also could you change `colSize` into `colWidth` and `getColSize` into `getColWidth` everywhere to be consistent?

**Comment by tristanguigue** [comment]
Ok that make sense see comments on implementation

**Pull request closed without merging**